README.md

```diff
@@ -5,11 +5,21 @@ This repository hosts the code used to generate the [keras.io](https://keras.io)
 ## Generating a local copy of the website
 
 ```
+pip install -r requirements.txt
 cd scripts
 python autogen.py make
 python autogen.py serve
 ```
 
+If you have Docker (you don't need the gpu version of Docker), you can run instead:
+
+```
+docker build -t keras-io . && docker run --rm -p 8000:8000 keras-io
+```
+
+It's going to take a while the first time, because it's going to pull the 
+image and the dependencies, but the second time, it's much faster.
```

Comment:
```suggestion
It will take a while the first time because it's going to pull the 
image and the dependencies, but on the next times it'll be much faster.
```

I don't think you even need to say this, as docker has a very good progress bar.
You can remove this paragraph entirely.

Reply:
It might not be obvious for people who are not used to docker. Thanks for the rewording, I'll commit your proposition. I still have the bug to fix about the comment tag. @pedro823 , did you manage to generate and serve the website locally by any chance (inside or outside docker)? Just to have another data point about the bug.

Reply:
I'll fork this branch and try it soon, will let you know :)

Reply:
I meant in master :), not necessarily this branch specifically. For me the bug is also happening in master.